reference-implementation/to-upstream-wpts/writable-streams/bad-strategies.https.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/service-workers/service-worker/resources/test-helpers.sub.js"></script>
+<script src="/streams/resources/test-initializer.js"></script>
```

Comment:
I think it would be better to use relative, as that makes it easier to move the streams directory around, in theory.